### DISCUSSION TITLE
Is there a way of getting the user info such as Email when using Social Logins with AppKit?

### DISCUSSION THREAD

**Discussion by GabrielaAlbu** [reown-com/appkit#3007] [Q&A] [answered]
Hi,

Using appkit with socials configured for google and apple for example, is there a function exposing the user Info, such as Email?

**Comment by jcr7467** [comment]
+1 I would also like to know this, UX is funky when I have to ask for email again

**Reply by KazanderDad** [reply]
+1 - This is a hard requirement for us to be able to use AppKit. Please answer.

**Reply by arein** [reply]
@KazanderDad how's this going - can you reach out to me so we can get you onboarded? https://github.com/orgs/reown-com/discussions/3007#discussioncomment-11444506

**Reply by arein** [reply]
> UX is funky when I have to ask for email again

@jcr7467 so you just need it on the frontend not on the backend?

**Reply by ethan-pacifica** [reply]
yes, is there a progress?

**Reply by arein** [reply]
> yes, is there a progress?

@aztonnish the hooks expose it on the front end 

Do you need it on the front end or backend ?

**Comment by hibagheadlabs** [comment]
Any solution?

**Reply by arein** [reply]
@hibagheadlabs can you reach out to me? https://github.com/orgs/reown-com/discussions/3007#discussioncomment-11444506

**Comment by arein** [comment]
@GabrielaAlbu @jcr7467 @hibagheadlabs yes we're launching this in alpha next week. Can you pls DM on Telegram so we can get you set up?

I'm "cyberdrk" on Telegram/Twitter

Looking forward !

**Comment by motiejunas** [comment]
Any update on this?

**Reply by arein** [reply]
yes. We launched our ["User Management"](https://docs.reown.com/cloud/user-management) feature that gives you access to `0x` addresses on the backend. Social/Email is next. https://github.com/orgs/reown-com/discussions/3007#discussioncomment-11444506 can you reach out to me so we can get you onboarded?

**Comment by hibagheadlabs** [comment, marked as answer]
Hi Derek, sorry for the delay in my feedback. I'm on holiday until the end
of February. However, I'll see if I can get online at some point and do a
test to confirm if the EMAIL is auto-filled correctly.
I just saw your updated documentation and it's great what you've done.
https://docs.reown.com/appkit/react/core/hooks
type EmbeddedWalletInfo {
user: {
username: string
email: string
},
accountType: 'eoa' | 'smartAccount',
authProvider: 'google' | 'apple' | 'facebook' | 'x' | 'discord' |
'farcaster' | 'github' | 'email',
isSmartAccountDeployed: boolean
}

this is exactly what i needed. I promise to have news as soon as possible.
great job!

regards,
Lucas

**Reply by dellwatson** [reply]
is this not available in react native yet ? @arein
